frontend/src/pages/SignupPage.js

```diff
@@ -48,31 +48,18 @@ const SignupPage = () => {
 
   const setSuccessDialog = () => {
     setIsDialogOpen(true)
-    setDialogTitle('Sign up Success')
-    setDialogMsg(
-      <>
-        <DialogContentText>
-          {'Your account has been successfully created.'}
-        </DialogContentText>
-        <DialogContentText>
-          {'You will now be redirected to the Log in page.'}
-        </DialogContentText>
-      </>
-    )
+    setDialogTitle('Account Created Successfully')
+    setDialogMsg('You will now be redirected to the Log in page.')
   }
 
   const setErrorDialog = (msg) => {
     setIsDialogOpen(true)
     setDialogTitle('Unable To Create Account')
-    setDialogMsg(
-      <>
-        <DialogContentText>{msg}</DialogContentText>
-      </>
-    )
+    setDialogMsg(msg)
   }
 
   const redirectButton = () => (
-    <Button className="font-inter" component={Link} to="/login">
```

Comment:
I was trying and realised that it is not possible to change the style of buttons in dialog just by using `font-inter` from Tailwind CSS. Hence, I removed it.